explore-analyze/visualize/charts/area-charts.md

```diff
@@ -0,0 +1,168 @@
+
+![Example Lens area chart compare periods](../../images/kibana-area-compare-periods.png)
+
+1. Create an area chart with a time-based **Horizontal axis** and your main metric on the **Vertical axis**, for example, `bytes`.
+2. From the three-dot menu in the upper-left of the layer panel, select **Duplicate layer**.
+3. From the duplicated layer, open the **Advanced** of the **Vertical axis** and set **Time shift** to `1w`. 
+   Check also [Compare differences over time](../lens.md#compare-data-with-time-offsets) for more details.
+4. Use a different color and set **Stacking** to **None** so areas overlay rather than stack.
+5. Update the legend to show “Current” and “Previous”.
+
+::::{tip}
+You can also compute the relative change using a formula, for example:  
+`(average(bytes) - average(bytes, shift='1w')) / average(bytes, shift='1w')`
+:::: 
+
+## Area chart settings [area-chart-settings]
+
+Customize your area chart to match the information you need and how you want it displayed.
+
+### Horizontal axis settings [horizontal-axis-settings]
+
+**Data**
+:   **Functions**: Allow you to group your data. For example, you can use `Date histogram` to group data points into time-based buckets, or `Filters` to divide values into predefined subsets.
+:   **Fields**: Determine which field from your data will be used for the horizontal axis.
+:   **Minimum interval**: Determine the smallest time bucket that your data will be grouped into (for example, seconds, minutes, hours, days).
+
+**Appearance**
+:   **Name**: By default, the chart uses the function or formula as title. It's a best practice to customize this with a meaningful title.
+
+### Vertical axis settings [vertical-axis-settings]
+
+**Data**
+:   To represent the metrics or values you want to visualize, you can use quick functions like `Average`, `Count`, `Percentile`, `Counter rate`, or create custom calculations with formulas. Refer to [](/explore-analyze/visualize/lens.md#lens-formulas) for examples, or to the {icon}`documentation` **Formula reference** available from Lens.
+
+    :::{include} ../../_snippets/area-vertical-axis-advanced-settings.md
+    :::
+
+**Appearance**
+:   Configure series-level options, including:
+   - **Name**: Customize the series label.
+   - **Value format**: Control how numeric values are displayed on your vertical axis and in tooltips.
+   - **Series color**: Determine the color of your data series in the visualization.
+   - **Axis side**: Determine which side of the chart the vertical axis appears on.
+
+### Breakdown settings [breakdown-settings]
+
+You can optionally split your series by a categorical field to create multiple stacked or overlapping areas. 
+
+## Area chart examples
+
+**Traffic by geographic region**
+:   Visualizing which geographic regions generate the most traffic:
+   - **Horizontal axis**: `@timestamp` (Date histogram)
+   - **Vertical axis**: `records`
+   - **Breakdown**: `geo.dest`
```

Comment:
Where did this data and these fields come from? I thought maybe `geo.dest` was an ECS field, but I don't see it [here](https://www.elastic.co/docs/reference/ecs/ecs-geo). Even if it is ECS, I'm not sure it's helpful to mention field names and the axis they map to. It would be more helpful to explain what the chart is showing and what data is on each axis/the breakdown so users can recreate with their own data.

Reply:
The format of the chart examples section follows a common pattern that we adopted in other chart types as well (Tables, Bar, Metric).